JS/Lesson24/goksel/script.js

```diff
@@ -0,0 +1,40 @@
+let formElement = document.getElementById("myForm");
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+formElement.addEventListener("submit", function (event) {
+
+    event.preventDefault();
+
+    let isFormCorrect = true;
+
+    // Variables for to check if email format is valid
+    let inputElement = document.getElementById("email");
+    let emailErrorElement = document.getElementById("email-error");
+
+    //Variables to change .active-form class after submitted
+    let formContainer = document.querySelector(".form-container");
+    let confirmContainer = document.querySelector(".confirm-container");
+
+    //Variable for to use user email on next form inside of the span content
```

Comment:
```suggestion
    // Variable for selecting user email on next form inside of the span content
```

Reply:
Normally, you should not add comments like this to your code. Instead, the name of the variable / function should tell you what you added as a comment.